python/paddle/fluid/contrib/utils/lookup_table_utils.py

```diff
@@ -159,8 +157,65 @@ def is_valid(var):
 
         return is_valid
 
-    def _load_lookup_table_vars(executor, dirname, main_program,
-                                lookup_table_vars):
+    io.load_vars(
+        executor,
+        dirname=dirname,
+        main_program=program,
+        predicate=_is_checkpoint_var(lookup_table_vars),
+        filename=None)
+
+
+def load_persistables_for_increment(dirname, executor, program,
```

Comment:
API should have comments for arguments and return type

Reply:
I have added comments for arguments and return type.